lib/tessel/update.js

```diff
@@ -94,7 +94,14 @@ Tessel.prototype.updateOpenWRT = function(image) {
                   return;
                 }
               });
-            });
+            })
+            .catch(function () {
+              // SHRUG. If the SSH session cut out, perhaps we can continue.
+              console.log('Maybe the update completed...? Waiting 30s');
```

Comment:
I would think that we should throw an error statement if anything might have gone wrong. Better to have folks re-update rather than incorrectly think their T2 is updated.